src/util/BuiltInFunctions.lua

```diff
@@ -17,11 +17,21 @@ local posix = require("posix.fnmatch")
 
 local BuiltInFunctions = {}
 
+function BuiltInFunctions.validateVariadicArgs(expectedLen, args)
+    if #args ~= expectedLen then
+        return error("Expected"..expectedLen.." arguments, but got "..#args)
+    end
+    for i=1,expectedLen do
+        if type(args[i])~="string" then
+            return error("Argument must be a string")
+        end
+    end
+    return nil
```

Comment:
I think this line can be removed since by default any function returns nil.

Reply:
Got it,fixed

Reply:
@Edmond-J-A Can you check it again, it seems it is still there.

Reply:
OK ， I push  it again.